package.json

```diff
@@ -2,9 +2,30 @@
   "name": "snabbdom",
   "version": "0.7.4",
   "description": "A virtual DOM library with focus on simplicity, modularity, powerful features and performance.",
-  "main": "snabbdom.js",
+  "type": "module",
+  "main": "./es/snabbdom.js",
```

Comment:
Why not use rollup here? This will break for everyone with a node less than 14.

Reply:
That's kind of the whole point of this pure es modules work. To remove the need for bundlers and simplify workflow. It's intentionally breaking with the old way, that's why this work isn't being done on master branch.

Reply:
If we provide as is in this branch (ES modules with `.js` extensions and extensions in import paths), then Node.js consumers could transpile it themselves. Whoever goes to the lengths of server side rendering or testing in Node.js instead of in a browser, should be able to solve this. And this is the future of packages, isn't it? Publish in the spec compliant form and let the consumer figure out the rest. No?